aws-storage-s3/src/main/java/com/amplifyframework/storage/s3/request/AWSS3StorageGetPresignedUrlRequest.java

```diff
@@ -26,13 +27,15 @@
  * @deprecated  this class is only constructed internally through deprecated transfer methods.
  */
 @Deprecated
+@InternalAmplifyApi
```

Comment:
We can't add this as it would cause a breaking change in forcing customer to add annotation to allow usage.

Reply:
annotation removed.